IFC4x3/Constants/v/VOID_2hj_ExWTf21wnmjHgzcx24.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="VOID_2hj_ExWTf21wnmjHgzcx24" Name="VOID" UniqueId="abb7e3bb-81da-4207-ac70-b51abd9bb084">
+	<Documentation>Representation of the concept of an identified discrete air filled geological feature, including caves and other voids.</Documentation>
+</DocConstant>
+
```

Comment:
IfcGeoScienceFeature from IfcTunnel will be covering this in another way.

Reply:
Agreed.